### PR TITLE
Command palette

### PR DESCRIPTION
**Pull request by zanemcca**
## Completed Items
- Create a command palette that allows users to interact with Sunsama without using the mouse
  - UI/UX of the palette
  - command shortcut to open palette

https://www.loom.com/share/c38354a2f62744349ea23c1f3d654723

### CODE REVIEW

**Comment by zanemcca** [comment]
[Integrate Command Palette](https://trello.com/c/IJuoTqXK/1-integrate-command-palette)

**Comment by priyadarshy** [comment]
Thanks for the demo video and update, I've gone ahead and merged this in. Travis and Bruno may check in on this tomorrow but I wanted to keep you unblocked. 

It's not uncommon for us to hop in and collaborate on a teammate's branch (like I did here). Let me know if that's an issue for your workflow!

**Comment by zanemcca** [comment]
I actually really appreciate the touchups you added so no disruption in my flow. 

You might notice that I am going a little rogue on master right now to just get deployments figured out.

**Comment by priyadarshy** [comment]
Awesome. As for going rogue on master, no issues here!

**Comment by zanemcca** [comment]
Sweet! FYI the application is now live at https://tp4000.zanemccaig.com/

File: src/App.tsx [hunk 1]
```diff
@@ -21,47 +22,49 @@ const channels = {
...
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a sapien orci. Integer efficitur nunc sem, tristique lacinia lorem egestas at. Sed iaculis purus non nisl aliquet, tempor efficitur risus ornare. Aenean sapien lectus, varius quis lacus eget, facilisis ultricies leo.',
-      channel: channels.default,
-      time: moment.duration('01:00:00'),
-    }, {
-      description: 'Nam malesuada massa odio.',
-      complete: true,
-    }]
+const today = moment()
+
+const schedules: IDailySchedule[] = [{
```
Comment (BrunoBernardino) [inline src/App.tsx]: Thanks for moving this out!

**Comment by priyadarshy** [comment]
> Sweet! FYI the application is now live at https://tp4000.zanemccaig.com/

Sick! It's loading up for me, nice work :)

**Pull request merged**